src/webextension/locales/en-US/manage.ftl

```diff
@@ -1,22 +1,22 @@
 toolbar-item-count = { $count ->
-   [one]   1 item
-  *[other] { $count } items
+   [one]   1 Entry
+  *[other] { $count } Entries
 }
 
-toolbar-add-item = Add Item
+toolbar-add-item = New Entry
 toolbar-go-home = Home
 toolbar-send-feedback = Feedback
 
-item-filter.placeholder = Filter…
+item-filter.placeholder = Search for an Entry
 
-item-summary-new-item = (new item)
-item-summary-no-title = (no title)
-item-summary-no-username = (no username)
+item-summary-new-item = New Entry
+item-summary-no-title = (No Entry Name)
+item-summary-no-username = (No Username)
```

Comment:
Not sure about title case here... Maybe we really just need some CSS to style these specially, like as italic or something?

Reply:
I've dealt with too much internationalization concerns elsewhere to feel confident with CSS directives here (e.g., Turkish dotless i).  Maybe I shouldn't be so concerned, but I'd rather start with the strings matching the mockups directly.

Reply:
That's fine; I was just thinking about greying them out a bit or something to indicate that they're not human-entered.

Reply:
The italics or color change seems like something we can follow up on ... this text matches the invision mockup + how I interpret the Photon guidelines.